### ISSUE TITLE
2.0 BETA

### ISSUE THREAD

**Issue by k1sul1** [anttiviljami/wp-libre-form#178]
As some of you might know, I've created a new repository for 2.0: https://github.com/libreform/libreform. Development of 2.0 is done there, and this repository turns obsolete as soon as 2.0 is production ready. It's not that yet, but it's pretty close. At this point, I'm fairly certain it won't `DROP TABLE wp_posts` or anything, but it has been tested by me, and me only. 

To become a beta tester, all you have to do is head over to the [Releases](https://github.com/libreform/libreform/releases) page, and download the latest release, and upload it to your WordPress instance. It would be nice if you could give me your thoughts about it afterwards. It would be great if you tried & used the APIs to build custom stuff and reported your experience with that. If you have forms made with an earlier version of WPLF, try copypasting them to the new version, reporting any troubles to me.

**TL;DR** WPLF 2.0 is in beta. It's better than 1.5. Much better. 

If you have any questions, you can send them here or send them to me with DM. 

### FAQ: 

**Do I get something from testing this?**

You get a 2.0 and you get to have a say on how some how it ends up. There's also plenty of #hacktoberfest PRs to go around if you're into t-shirts or trees. 

**How does X work**
I don't want to write a longer wall of text here describing the key changes, how it works, etc. because [you can find all that in the documentation](https://github.com/libreform/libreform/blob/master/README.md), which is currently in the form of markdown files in the docs folder. 

If you can't find it, or it isn't there, drop me a DM anywhere.
